src/components/ui/Icons/User.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { withTheme } from 'styled-components';
+
+const User = ({ theme, ...props }) => (
+  <svg height="36px" viewBox="0 0 1000 1000" width="36px" {...props}>
+    <g>
+      <path
+        d="M500,10C228.8,10,10,228.8,10,500c0,271.3,218.8,490,490,490c271.3,0,490-218.8,490-490C990,228.8,771.3,10,500,10z M804.5,822C794,657.5,619,613.8,619,613.8s106.8-71.8,68.3-217c-19.3-75.3-91-131.3-189-131.3c-98,0-169.7,56-189,131.3c-38.5,145.2,68.2,217,68.2,217S202.5,652.3,192,822C109.7,739.7,57.3,626,57.3,500C57.3,255,255,57.3,500,57.3C745,57.3,942.8,255,942.8,500C942.8,626,890.2,739.7,804.5,822z"
+        fill={theme.text}
+      />
+    </g>
+  </svg>
+);
```

Comment:
I've just minified this SVG through [svgr](https://react-svgr.com/playground/) tool

```javascript
const User = ({ theme, ...props }) => (
    <svg height={36} viewBox="0 0 1000 1000" width={36} {...props}>
      <path d="M500 10C228.8 10 10 228.8 10 500c0 271.3 218.8 490 490 490 271.3 0 490-218.8 490-490S771.3 10 500 10zm304.5 812C794 657.5 619 613.8 619 613.8s106.8-71.8 68.3-217c-19.3-75.3-91-131.3-189-131.3s-169.7 56-189 131.3c-38.5 145.2 68.2 217 68.2 217S202.5 652.3 192 822C109.7 739.7 57.3 626 57.3 500 57.3 255 255 57.3 500 57.3S942.8 255 942.8 500c0 126-52.6 239.7-138.3 322z" />
    </svg>
);
```